pnpm-lock.yaml

```diff
@@ -6609,6 +7208,33 @@ packages:
       regexparam: 3.0.0
     dev: false
 
+  /wrangler@3.24.0:
+    resolution: {integrity: sha512-jEnqpY+9/J4VPjtuEnS2lhCPXkvbDClnMalSWaRxSx+1tiTWMJhMjtK9oyXLdO+ZUf9Q4LvFTYSPm8O1uwmnxQ==}
+    engines: {node: '>=16.17.0'}
+    hasBin: true
+    dependencies:
+      '@cloudflare/kv-asset-handler': 0.2.0
+      '@esbuild-plugins/node-globals-polyfill': 0.2.3(esbuild@0.17.19)
+      '@esbuild-plugins/node-modules-polyfill': 0.2.2(esbuild@0.17.19)
+      blake3-wasm: 2.1.5
+      chokidar: 3.5.3
+      esbuild: 0.17.19
```

Comment:
think it'd be possible to upgrade `esbuild` to 0.19? that would reduce the number of versions of esbuild that we pull in

Reply:
No unfortunately we can't upgrade `esbuild` to 0.19 😓 
See: https://github.com/evanw/esbuild/issues/3337